src/components/ui/combobox.tsx

```diff
@@ -83,6 +86,7 @@ export function ComboBox({
 
   const [open, setOpen] = React.useState(false)
```

Comment:
I dont think we have any style decisions regarding this, I know I for one prefer a import of useState, instead of React.useState(...).

Reply:
I agree, prefer it that way myself. Guess I just went along with what was already implemented and never thought about changing the code. Will fix.